pkg/controller/jenkinsimage/jenkinsimage_controller.go

```diff
@@ -2,28 +2,29 @@ package jenkinsimage
 
 import (
 	"context"
-
-	jenkinsv1alpha1 "github.com/redhat-developer/openshift-jenkins-operator/pkg/apis/jenkins/v1alpha1"
-	corev1 "k8s.io/api/core/v1"
-	//appsv1 "github.com/openshift/api/apps/v1"
-	imagev1 "github.com/openshift/api/image/v1"
 	buildv1 "github.com/openshift/api/build/v1"
+	imagev1 "github.com/openshift/api/image/v1"
+	jenkinsv1alpha1 "github.com/redhat-developer/openshift-jenkins-operator/pkg/apis/jenkins/v1alpha1"
+	j "github.com/redhat-developer/openshift-jenkins-operator/pkg/controller/controllerutil"
 	"k8s.io/apimachinery/pkg/api/errors"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/types"
+	"reflect"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/controller"
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
-	"sigs.k8s.io/controller-runtime/pkg/handler"
-	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
-	"sigs.k8s.io/controller-runtime/pkg/source"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
 )
 
+const ImageStreamTagKind = "ImageStreamTag"
+const DockerImageKind = "DockerImage"
 
-const registryHostname = "image-registry.openshift-image-registry.svc:5000"
+const DefaultRegistryHostname = "image-registry.openshift-image-registry.svc:5000"
+const DefaultImageNamespace = "openshift"
+const DefaultJenkinsBaseImage = "jenkins" + ":" + "2"
+const DefaultImageStreamTag = "latest"
 
 var log = logf.Log.WithName("controller_jenkinsimage")
```

Comment:
is this should be jenkinsimage_controller in place of controller_jenkinsimage ??

Reply:
fixed